pkg/probe/bgp_neighbors_ipv4.go

```diff
@@ -0,0 +1,66 @@
+package probe
```

Comment:
Maybe keep ipv4 and ipv6 in the same file? Seems a bit much to split them up given it is only one function in the file.

Reply:
In that case you can also share the BGPNeighbor structure (unless they differ?) between IPv4 and IPv6.

Reply:
Agree, I'll combined them into one file